.coveragerc

```diff
@@ -7,9 +7,8 @@ branch = True
 plugins = Cython.Coverage
 include =
     cf_units/*
-omit = 
+omit =
     setup.py
-    versioneer.py
     cf_units/_version.py
     cf_units/etc/*
     cf_units/tests/*
```

Comment:
@trexfeathers If we're in the business of minimising the number of configuration files, then `.coveragerc` can also be added to the `setup.cfg` :+1:

Reply:
a264e37069cb721ad1928cfe83419282214ade43